### DISCUSSION TITLE
Your House Modes

### DISCUSSION THREAD

**Discussion by MarcHagen** [frenck/home-assistant-config#124] [General]
Hi Frenck!
Awesome config and love the way you have set up your house modes.
I was wondering if you perhaps have an older version here than you are running?

In the `automations/house/mode.yaml` you point to `mode_X_house`
https://github.com/frenck/home-assistant-config/blob/0cb55f53b28943cf254e6b81a1b9fa94d637c030/automations/house/mode.yaml#L30-L35

But most automations are using `house_mode_X`
https://github.com/frenck/home-assistant-config/blob/0cb55f53b28943cf254e6b81a1b9fa94d637c030/automations/kitchen/water_heater.yaml#L26-L29
https://github.com/frenck/home-assistant-config/blob/0cb55f53b28943cf254e6b81a1b9fa94d637c030/automations/hall/lights.yaml#L25-L29

**Comment by frenck** [comment]
Yeah my config here isn't up to date with anything right now

**Comment by essel-dev** [comment]
I learned a lot from your config @frenck !

**Comment by jhonpaul555** [comment]
Your House Modes #124 shares some really helpful ideas for homeowners who want to understand their property better. I especially like how easy it has become to access information through the https://lacountyassessors.org when researching home values and records. Having reliable property details available can make planning, buying, or managing a home much easier. This is definitely useful information for anyone interested in real estate.
